backend/tracim_backend/tests/functional/test_webdav.py

```diff
@@ -223,3 +275,276 @@ def test_functional__webdav_access_to_workspace__workspace_not_exist(self) -> No
+        )
+        transaction.commit()
+
+        self.testapp.authorization = (
+            'Basic',
+            (
+                'admin@admin.admin',
+                'admin@admin.admin'
+            )
+        )
+        self.testapp.get('/workspace1', status=200)
+        self.testapp.get('/workspace1/file.txt', status=404)
+
+    def test_functional__webdav_access_to_subdir_content__ok__nominal_case(self) -> None:
+        dbsession = get_tm_session(self.session_factory, transaction.manager)
+        admin = dbsession.query(User) \
+            .filter(User.email == 'admin@admin.admin') \
+            .one()
+        workspace_api = WorkspaceApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+            show_deleted=True,
+        )
+        workspace = workspace_api.create_workspace('workspace1', save_now=True)  # nopep8
+        api = ContentApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+        )
+        folder = api.create(
+            content_type_list.Folder.slug,
+            workspace,
+            None,
+            'folder',
+            do_save=True,
+            do_notify=False,
+        )
+        with dbsession.no_autoflush:
+            file = api.create(
+                content_type_list.File.slug,
+                workspace,
+                folder,
+                filename='file.txt',
+                do_save=False,
+                do_notify=False,
+            )
+            api.update_file_data(
+                file,
+                'file.txt',
+                'text/plain',
+                b'test_content'
+            )
+            api.save(file)
+        transaction.commit()
+
+        self.testapp.authorization = (
+            'Basic',
+            (
+                'admin@admin.admin',
+                'admin@admin.admin'
+            )
+        )
+        self.testapp.get('/workspace1', status=200)
+        self.testapp.get('/workspace1/folder', status=200)
+        self.testapp.get('/workspace1/folder/file.txt', status=200)
+
+    def test_functional__webdav_access_to_subdir_content__ok__special_characters(self) -> None:
+        dbsession = get_tm_session(self.session_factory, transaction.manager)
+        admin = dbsession.query(User) \
+            .filter(User.email == 'admin@admin.admin') \
+            .one()
+        workspace_api = WorkspaceApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+            show_deleted=True,
+        )
+        workspace = workspace_api.create_workspace('/?\#*', save_now=True)  # nopep8
+        api = ContentApi(
+            current_user=admin,
+            session=dbsession,
+            config=self.app_config,
+        )
+        folder = api.create(
+            content_type_list.Folder.slug,
+            workspace,
+            None,
+            '/?\#*',
```

Comment:
probably add a comment "create a file named '?\#*' by the way, the backslash is not interpreted with the # ?

Reply:
\ is interpreted correctly it seems that \ is interpreted in a specific manner just in some specific case. escaping it work the same way, so i do escape it now.